### ISSUE TITLE
Press Keys and Write Text do nothing at all

### ISSUE THREAD

**Issue by desrod** [timothycrosley/streamdeck-ui#80]
I was hoping that the Press Keys and Write Text would allow me to send keys like F1, Ctrl and other meta keys and Write Text would send arbitrary text into the app being launched by the button, but these appear to be empty placeholders in the UI, with no code behind them to support those actions. 

When, if at all, will those two options be supported, or can they be removed from the UI, so they're not confusing to users like myself who tried, unsuccessfully, to use them?

**Comment by desrod** [comment]
Adding to my previous.. I can create a new target to launch Sublime Text (subl), and use the 'Press Keys' to pass 'Ctrl+N' to create a new window, and then in 'Write Text' put 'This is a test'. 

When I press the button, Sublime launches. A new window is _not_ opened, the text is _not_ added to the open document in the editor. 

Pressing the configured button a second time, launches a new subl instance in its own window. The configured 'Write Text' value is not sent to that open window. 

Pressing the configured button a _third_ time, launches a new subl instance in its own (third) window. The configured text is then sent into the first and second windows simultaneously, but not the third window. 

In other words, window 1 gets 'This is a test' appended to wherever the cursor is, and window 2 gets 'This is a test' as its only entry in the new, blank editor session. 

My expectation is that the text would be appended to the window launched with the _first_ button press, after opening a new document (Ctrl+N) from the 'Press Keys' option. In other words, chaining the options together: 

1. Launch subl
2. Ctrl+N
3. Send 'This is a test' into the one-and-only instance of Sublime Text

Is that not how these are designed to work?

**Comment by dodgyrabbit** [comment]
There are a couple of fixes in the pipeline. Likely the solution is not to use seperate actions, but to launch sublime and  pass the text as part of std in using the command function. There are some issues with the current release build, once the new update is published, please give it a try and reopen if it is still an issue.
